aten/src/ATen/native/native_functions.yaml

```diff
@@ -9002,15 +9002,22 @@
   dispatch:
     DefaultBackend: linalg_cholesky_out
 
-# torch.linalg.det, alias for torch.det
 - func: linalg_det(Tensor self) -> Tensor
   python_module: linalg
   variants: function
+  dispatch:
+    DefaultBackend: linalg_det
+
+- func: linalg_det.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  python_module: linalg
+  dispatch:
+    DefaultBackend: linalg_det_out
 
+# torch.det, alias for torch.linalg.det
 - func: det(Tensor self) -> Tensor
```

Comment:
Does the autograd for det work after these changes? I always get confused about how math and defaultbackend relate in these cases.

Reply:
yes, that's my understanding as well. Moreover now it has been renamed to `CompositeAutogradImplicit` and it's the default, so I fixed my `native_functions.yml` accordingly